### ISSUE TITLE
OSA Meeting Friday April 23rd 2021?

### ISSUE THREAD

**Issue by mattodd** [opensourceantibiotics/Series-2-Diarylimidazoles#69]
Hi all - meeting today? As far as I can tell:

0) We're awaiting crucial tox data from Alex.

1) we are still working on the action items from last time (#68) <-- it's on me to upload recordings, will do.

2) @danaklug posted a really nice summary of chemical next steps [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/66#issuecomment-822494145) that includes the molecules @flavioemery volunteered to make and some targets for @Giada-chem, and @MFernflower reminded us that we need to purchase a couple of amides.

3) @edwintse is looking into purifying the Hypha metabolite, and when we have it it can be added to @loriferrins ' compounds for evaluation by Paul Stapleton (who is happy to do this, just spoke with him).

4) Dana is shipping stuff to DNDi, I'm writing a newsletter (though I have to write something for World Malaria Day first..!)

So I suggest we skip this week, gain 30 minutes of individual quality science time (always a pleasure, right @drc007?) and reconnect next week. Any howls of disappointment?

**Comment by drc007** [comment]
Fine with me

**Comment by flavioemery** [comment]
Perfect!

**Comment by danaklug** [comment]
Fine by me, I'll be using the extra time to weigh out our compounds going to DNDi! Was hoping to get them out by end of this week but looks like it will be early next week instead - I'll post an update when they've shipped.

**Comment by MFernflower** [comment]
I hereby propose we call these going forward  Pseudomeetings

Thoughts everyone?

**Comment by MFernflower** [comment]
@mattodd I think we did get some preliminary data re tox: https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Cytotox/Tox2.jpg

**Comment by MFernflower** [comment]
@mattodd @danaklug Assuming the data is correct above - perhaps this might be worth exploring?

![N%u2010(cyclobutylmethyl)%u2010N%u2010(4%u2010fluorophenyl)%u20102%u2010(pyridin%u20102%u2010 yl)imidazo 1,2%u2010a pyridin%u20103%u2010amine](https://user-images.githubusercontent.com/3164942/115964389-4ca73700-a4f2-11eb-9560-ca840d27f034.png)

**Comment by lferrins** [comment]
@MFernflower we have the oxetane analog up for purification so maybe hold off and wait for that one?

**Comment by MFernflower** [comment]
That's fantastic!

**Comment by MFernflower** [comment]
@lferrins Do open a github issue for that particular compound - i bet it will be quite an interesting gem to add to our SAR tables

**Comment by MFernflower** [comment]
@lferrins Do open a github issue for that particular compound once it's purified - I bet it will be quite the interesting gem to add to our SAR tables

**Comment by lferrins** [comment]
@MFernflower once it is ready to ship I will open a new issue!
